src/test/java/seedu/manager/command/ListCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.manager.command;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.manager.event.Event;
+import seedu.manager.event.EventList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class ListCommandTest {
+
+    private ListCommand listCommand;
+
+    @BeforeEach
+    public void setUp() {
+        EventList eventList = new EventList();
+
+        eventList.addEvent(new Event("Event 1", "2024-10-10 10:00", "Venue A"));
+        eventList.addEvent(new Event("Event 2", "2024-11-11 12:00", "Venue B"));
+
+        listCommand = new ListCommand();
+        listCommand.setData(eventList);
+    }
+
+    @Test
+    public void testExecute() {
```

Comment:
I think the name of this test could be changed to follow the format <feature>_<test scenario>_<expected outcome>, as per the coding standard